### ISSUE TITLE
Buy New FitBit Trackers for Sleep Tracking (Research)

### ISSUE THREAD

**Issue by nelsonic** [dwyl/sleep#9]
FirBit are still the most popular activity/sleep tracking wearables.
Which means we need to be _testing_ the device(s) for https://github.com/dwyl/learn-apple-watch-development/issues/35
And ...
![ines-miss-fitbit](https://cloud.githubusercontent.com/assets/194400/25997074/5a543e2c-3712-11e7-8bcb-94ef4d1d63fa.png)

@iteles please: 
+ [ ] decide _which_ FitBit you want and 
+ [ ] order it **today**. 

thanks! 👍

**Comment by iteles** [comment]
Ah, I have actually been purposefully _not_ doing this since we got the Apple Watches.
We're creating an app for the Apple Watch and as such, I want to be wearing mine.

If I buy a fitbit (which all come with watch faces and timers), I'm going to naturally stop wearing my Apple Watch (as above, I don't want this to happen). This is why I bought a [Bellabeat Leaf](https://webshop.bellabeat.com/products/leaf-urban-silver-edition) (on sale), which also has ok sleep tracking (for individual days) but the strap broke. Rather than buy a fitbit, I'm buying a new Leaf strap 👍 

I have used and loved the fitbit extensively and can happily write up a detailed blog post on it without a test device 👍
